packages/badge/Badge.test.tsx

```diff
@@ -1,18 +1,51 @@
-import { Badge } from './Badge';
-
+import { Badge } from './index';
 import { h, mount } from 'bore';
-import { emit } from 'skatejs';
 
 describe( Badge.is, () => {
 
   it( 'should render', () => {
 
-    console.warn('Missing tests for Badge!');
-
     return mount(
       <Badge/>
-    ).wait();
+    ).wait((element) => element.has('c-badge') );
 
   });
 
+  it( 'should render rounded', () => {
```

Comment:
it has nothing to do with testing asynchronous code. That's implementation detail of bore that `wait` returns Promise and that mocha is able to consume it.  that `mount` call is just "another" `before` hook specific to bore et all.

I want to have there assertions, to be consistent with all testing, although sometimes it will be on line more.

```
 it( 'should render intrinsic element', () => {
   return mount(
     <bl-badge></bl-badge>
   ).wait((element) => {
       const actual = element.has('c-badge'); 
       const expected = true;
       expect( actual ).to.equal( expected );
    });
  
});
```